plugins/scaffolder-backend/api-report.md

```diff
@@ -444,6 +444,14 @@ export function fetchContents({
   outputPath: string;
 }): Promise<void>;
 
+// @public @deprecated
+export type GetTokenAndEntityRefFromRequestFunction = (
```

Comment:
On second thought I think this should be async so that the user can be retrieved from an external system.